internal/store/ingress.go

```diff
@@ -136,6 +136,26 @@ var (
 				}
 			}),
 		},
+		{
+			Name: "kube_ingress_tls",
+			Type: metric.Gauge,
+			Help: "Ingress TLS host and secret information.",
+			GenerateFunc: wrapIngressFunc(func(i *v1beta1.Ingress) *metric.Family {
+				ms := []*metric.Metric{}
+				for _, tls := range i.Spec.TLS {
+					for _, host := range tls.Hosts {
+						ms = append(ms, &metric.Metric{
+							LabelKeys:   []string{"tls_host", "secret_name"},
```

Comment:
this should be `secret` instead of `secret_name` according to the convention (this allows for easier joining of resources)

Reply:
It is done